voltixApp/VoltixApp/Views/Vault/VaultPairDetailView.swift

```diff
@@ -55,7 +55,11 @@ struct VaultPairDetailView: View {
                     .frame(maxWidth: .infinity, alignment: .center)
                 
                 ForEach(devicesInfo, id: \.Index) { device in
-                    VaultPairDetailCell(title: device.Signer, description: .empty)
+                    if device.Signer == Utils.getLocalDeviceIdentity() {
```

Comment:
This is not correct, instead you should compare `device.Signer` , `vault.localPartyID`

Reply:
Changing now